go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/spf13/afero v1.1.2
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.9.0
```

Comment:
Can you fix this @grafana/k6-browser unless you need some new functionality in it

Reply:
No, we don't need the new functionality, but if it doesn't break the k6-core functionality, is there no harm in upgrading it? It's only used for tests.

Reply:
Is there a strong aversion to this upgrade?